lib/recipe.rb

```diff
@@ -0,0 +1,11 @@
+class Recipe
+  attr_reader :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def ingredients_required
```

Comment:
Same thing here. Anytime we want to accumulate or keep track of data for an instance, storing it as an instance variable is the best course.